examples/rbd/pvc-raw-block-clone.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: raw-block-pvc-clone
 spec:
   storageClassName: csi-rbd-sc
+  volumeMode: Block
```

Comment:
this should be squashed into the commit that adds this file

Reply:
yeah. sure.